StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -79,12 +79,12 @@ final class FinancialConnectionsAPIClient {
         return promise
     }
 
-    static func encodeAsParameters(_ value: any Encodable) throws -> [String: Any] {
+    static func encodeAsParameters(_ value: any Encodable) throws -> [String: Any]? {
```

Comment:
Could this have any unintended consequences somewhere else?

Reply:
Nope! This fixes an issue where we we're sending `[:]` when all the values were' nil or empty!